config/routes.rb

```diff
@@ -14,6 +14,9 @@
     root to: 'home#index'
     get "splash", to: "home#splash"
 
+    # email confirmation link
+    get 'confirm', to: 'registrations/placement_requests#create'
```

Comment:
Is there a reason to not do

```ruby
get 'confirm/:uuid', to: 'registrations/placement_requests#create', as: 'email_confirmation'
```

This has two benefits
1. We are actually naming the URL
2. We are avoiding the user needing to understand the semantics of query params - most are much more familiar with urls with slashes?

Obviously this is an edge case and its assumed the user will click rather than type the link out but doesn't seem a reason to not do it